db/log_format.h

```diff
@@ -39,8 +39,11 @@ enum RecordType {
   // User-defined timestamp sizes
   kUserDefinedTimestampSizeType = 10,
   kRecyclableUserDefinedTimestampSizeType = 11,
+
+  // Reserve kRecordTypeSafeIgnoreMask = 32
 };
-constexpr int kMaxRecordType = kRecyclableUserDefinedTimestampSizeType;
+const int kRecordTypeSafeIgnoreMask = 1 << 5;
+constexpr int kMaxRecordType = kRecordTypeSafeIgnoreMask + 5;
```

Comment:
It will soon to be overwritten to be the last entry type in WAL that is greater than kRecordTypeSafeIgnoreMask in my upcoming PR.